viz.Rmd

```diff
@@ -569,23 +820,115 @@ for just how accurate all the experiments were overall. For example, how accurat
+  ggtitle("Default bin width (bins = 30)")
+
+morley_hist_big <- ggplot(morley_rel, aes(x = relative_accuracy, fill = as_factor(Expt))) +
+  geom_histogram(position = "identity", binwidth = 0.1) +
+  facet_grid(rows = vars(Expt)) +
+  geom_vline(xintercept = 0, linetype = "dashed", size = 1.0) +
+  labs(x = "Relative Accuracy (%)", y = "# Measurements", fill = "Experiment ID") +
+  theme(legend.position = "none") +
+  ggtitle( "binwidth = 0.1")
+
+morley_hist_med <- ggplot(morley_rel, aes(x = relative_accuracy, fill = as_factor(Expt))) +
+  geom_histogram(position = "identity", binwidth = 0.01) +
+  facet_grid(rows = vars(Expt)) +
+  geom_vline(xintercept = 0, linetype = "dashed", size = 1.0) +
+  labs(x = "Relative Accuracy (%)", y = "# Measurements", fill = "Experiment ID") +
+  theme(legend.position = "none") +
+  ggtitle("binwidth = 0.01")
+
+morley_hist_small <- ggplot(morley_rel, aes(x = relative_accuracy, fill = as_factor(Expt))) +
+  geom_histogram(position = "identity", binwidth = 0.001) +
+  facet_grid(rows = vars(Expt)) +
+  geom_vline(xintercept = 0, linetype = "dashed", size = 1.0) +
+  labs(x = "Relative Accuracy (%)", y = "# Measurements", fill = "Experiment ID") +
+  theme(legend.position = "none") +
+  ggtitle("binwidth = 0.001")
+
+plot_grid(morley_hist_default, 
+          morley_hist_small, 
+          morley_hist_med, 
+          morley_hist_big, 
+          ncol = 2)
+```
+
+
+
+#### Adding layers to a `ggplot` plot object {-}
+
+What if we want to add onto a `ggplot` plot object later in our analysis?
+Do we need to copy and paste the code we wrote previously 
+and then add on our new layers after that? No! 
+If we stored our `ggplot` plot as a named object
+using the assignment symbol (`<-`) then we can call that object 
+and add on to it using the `+` operator.
+For example, if we wanted to add a title to the last plot we created, 
+named `morley_rel`, we can call the name of that plot, then the `+` operator, 
+and then add our code to create a title for the plot 
+(using the `ggtitle` function):
+
+```{r 03-data-morley-hist-addlayer, warning = FALSE, message = FALSE, fig.height = 7, fig.cap = "Histogram of relative accuracy split vertically by experiment with a descriptive title highlighting the take home message of the visualization."}
+morley_hist_title <- morley_hist +
+  ggtitle("Morley's speed of light experiments \n were accurate to about 0.05%")
+morley_hist_title
+```
+
+> **Note:** 
+> Good data visualizations titles clearly communicate 
+> the take home message to the audience. 
+> Typically, 
+> that is the answer to the question you posed before making the visualization.
 
 ## Explaining the visualization
-#### *Tell a story*
+#### *Tell a story* {-}
 
 Typically, your visualization will not be shown entirely on its own, but rather it will be part of a larger presentation.
 Further, visualizations can provide supporting information for any aspect of a presentation, from opening to conclusion.
```

Comment:
space between colon and "story" in this line: "Regardless of where it appears, a good way to discuss your visualization is as a story :"